addons/armaos/functions/fnc_computer_addActionTurnOff.sqf

```diff
@@ -0,0 +1,85 @@
+/**
+ * Adds the "turn off" ACE interaction for a given computer object. Computer texture changes accordingly if silent mode is false.
+ *
+ * Arguments:
+ * 1: Computer <OBJECT>
+ * 2: Silent Switch <BOOL>
+ *
+ * Results:
+ * None
+ */
+
+params ["_computer", "_silent"];
+
+_powerState = _computer getVariable 'AE3_powerState';
+
+_turnOffTime = 0;
+
+switch (_powerState) do
+{
+	case 1: { _turnOffTime = 15; };
+	case 2: { _turnOffTime = 10; };
+	default {};
+};
```

Comment:
why not:
```sqf
if(_powerState == 1) then
{
 _turnOffTime = 15;
} else
{
 _turnOffTime = 10;
};
```

Reply:
Sure, I will change that.

Reply:
changed in various files/places in commit https://github.com/y0014984/Advanced-Equipment/pull/90/commits/351ca2f7548ca9717ff26f606317f2c7e56c0763